README.md

```diff
@@ -86,6 +86,9 @@ Add `@nuxtjs/apollo` to `modules` section of `nuxt.config.js`
         // required  
         httpEndpoint: 'http://localhost:4000',
         // optional
+        // override HTTP endpoint in browser only
+        browserHttpEndpoint: '/graphql',
```

Comment:
Have you actually tried making a request with this set to `/graphql`. As far as I remember, apollo client won't work with relative URLs.

(I know this is just an example but still should have invalid examples.)

Reply:
Also happy to provide a different example such as `http://localhost:3000`.